### ISSUE TITLE
Remove descriptions from Introspection schema

### ISSUE THREAD

**Issue by frederikhors** [dotansimha/graphql-code-generator#4731]
**Is your feature request related to a problem? Please describe.**

Can we remove `descriptions` from introspection schema using the plugin?

**Describe the solution you'd like**

Something like they do here: https://github.com/graphql/graphql-js/blob/v15.3.0/src/utilities/getIntrospectionQuery.js:

```js
export type IntrospectionOptions = {|
  // Whether to include descriptions in the introspection result.
  // Default: true
  descriptions?: boolean,

  // Whether to include `specifiedByUrl` in the introspection result.
  // Default: false
  specifiedByUrl?: boolean,

  // Whether to include `isRepeatable` field on directives.
  // Default: false
  directiveIsRepeatable?: boolean,

  // Whether to include `description` field on schema.
  // Default: false
  schemaDescription?: boolean,
|};
```

**Comment by dotansimha** [comment]
Thank you @frederikhors , that's a great idea.
Added this in: https://github.com/dotansimha/graphql-code-generator/pull/4810

**Comment by dotansimha** [comment]
Fixed in `@graphql-codegen/introspection@1.18.0`
